### PR TITLE
CDS-28 Create boilerplate for non-landing pages

### PR DESCRIPTION
**Pull request by Liannus**
Changelog:
- Added boilerplate for formatting of non-landing pages

### CODE REVIEW

File: src/components/ContactLinks/ContactLinks.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.contact-links {
+  .title {
+    font-size: 2.5em;
+    font-weight: 350;
+  }
+
+  .text {
+    font-size: 1.5em;
+    font-weight: 300;
+    word-break: break-word;
```
Comment (forkball) [inline src/components/ContactLinks/ContactLinks.scss]: We should probably use tailwind for word-break

File: src/components/ContactLinks/ContactLinks.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  .textIcon {
+    font-size: 1.5em;
+    margin-right: 0.2em;
+    font-weight: 300;
+    padding: 0;
```
Comment (forkball) [inline src/components/ContactLinks/ContactLinks.scss]: Padding should be done with tailwind

File: src/components/ContactLinks/ContactLinks.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  .linebreak {
+    height: 3px;
```
Comment (forkball) [inline src/components/ContactLinks/ContactLinks.scss:21]: Should be down with tailwind

File: src/components/ContactLinks/ContactLinks.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      transparent,
+      $color-off-white,
+      $color-off-white,
+      transparent
+    );
+  }
+
+  .icon {
+    font-size: 3em;
+    margin: 0.2em 0.5em;
```
Comment (forkball) [inline src/components/ContactLinks/ContactLinks.scss]: Should be done with tailwind

**Review by forkball — COMMENTED** [review]
Some CSS stylings should be done with tailwind, otherwise looks good.

File: src/components/ContactLinks/ContactLinks.scss (hunk 1 above)
Comment (Liannus) [reply src/components/ContactLinks/ContactLinks.scss]: Resolved

File: src/components/ContactLinks/ContactLinks.scss (hunk 2 above)
Comment (Liannus) [reply src/components/ContactLinks/ContactLinks.scss]: resolved

File: src/components/ContactLinks/ContactLinks.scss (hunk 3 above)
Comment (Liannus) [reply src/components/ContactLinks/ContactLinks.scss:21]: Tailwind doesn't go small enough, `h-1`: 0.25rem looks odd imo.

File: src/components/ContactLinks/ContactLinks.scss (hunk 4 above)
Comment (Liannus) [reply src/components/ContactLinks/ContactLinks.scss]: Resolved

File: src/components/ContactLinks/ContactLinks.scss (hunk 3 above)
Comment (forkball) [reply src/components/ContactLinks/ContactLinks.scss:21]: Hmm fair enough then.

File: src/components/Card/Card.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
+@import "styles/colors.scss";
+
+.card {
+  color: $color-off-white;
+  font-family: "Open sans";
+
+  position: relative;
```
Comment (forkball) [inline src/components/Card/Card.scss]: Should be done with tailwind, missed this on my first look.

File: src/components/Card/Card.scss [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  z-index: 2;
+
+  .inner-container {
+    background-color: $color-dark-grey;
+    margin: -0.75rem;
```
Comment (forkball) [inline src/components/Card/Card.scss]: Also should be done with tailwind.

File: src/components/Card/Card.scss [hunk 7]
```diff
@@ -0,0 +1,14 @@
+@import "styles/colors.scss";
+
+.card {
+  color: $color-off-white;
+  font-family: "Open sans";
+
+  position: relative;
+  z-index: 2;
```
Comment (forkball) [inline src/components/Card/Card.scss]: Can be done in tailwind, but should it be?

File: src/components/ContactLinks/ContactLinks.scss [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  .text {
+    font-size: 1.5em;
+    font-weight: 300;
+  }
+
+  .textIcon {
+    font-size: 1.5em;
+    margin-right: 0.2em;
```
Comment (forkball) [inline src/components/ContactLinks/ContactLinks.scss]: Should be done in tailwind, missed this on first look.

**Review by forkball — CHANGES_REQUESTED** [review]
Sorry, missed some stuff on my first look.

File: src/components/Card/Card.scss (hunk 5 above)
Comment (Liannus) [reply src/components/Card/Card.scss]: resolved

File: src/components/Card/Card.scss (hunk 6 above)
Comment (Liannus) [reply src/components/Card/Card.scss]: can't do negative margins with tailwind.

File: src/components/Card/Card.scss (hunk 7 above)
Comment (Liannus) [reply src/components/Card/Card.scss]: yep, resolved

File: src/components/ContactLinks/ContactLinks.scss (hunk 8 above)
Comment (Liannus) [reply src/components/ContactLinks/ContactLinks.scss]: resolved

File: src/components/Card/Card.scss (hunk 5 above)
Comment (Liannus) [reply src/components/Card/Card.scss:7]: my b, resolved

File: src/components/Card/Card.scss (hunk 6 above)
Comment (Liannus) [reply src/components/Card/Card.scss]: my b, resolved

**Comment by Liannus** [comment]
Also changed dynamic-fonts to `dynamic-header` and `dynamic-text` with slightly different breakpoints that I think work better and updated some fonts on cards and headers that didn't have them already.

**Review by forkball — APPROVED** [review]

**Pull request merged**